docs/contentSharing.md

```diff
@@ -1,8 +1,8 @@
-# Question sharing
+# Content sharing
 
 !!! info
-
-    **Question sharing is currently in beta.** If your course is hosted on the official PrairieLearn server, you may request that question sharing be turned on for your course. If you administer your own PrairieLearn server, you should _not_ use question sharing at this time. In the future, question sharing may be supported across PrairieLearn servers, so turning on question sharing on unofficial servers at this point in time may lead to naming conflicts in the future.
+Viewing and consuming publicly shared questions and assessments is now general access.  
+ Sharing your own questions and assessments is still a Beta feature. If your course is hosted on the official PrairieLearn server, you may request that question sharing be turned on for your course. If you administer your own PrairieLearn server, you should _not_ use question sharing at this time. In the future, question sharing may be supported across PrairieLearn servers, so turning on question sharing on unofficial servers at this point in time may lead to naming conflicts in the future.
```

Comment:
What went on with the formatting here? Will suggest more changes once this is resolved.

Reply:
@SethPoulsen I just saw your other comment about this. These are called "admonitions". See https://squidfunk.github.io/mkdocs-material/reference/admonitions/?h=admoniti#usage for the proper formatting.

Reply:
thanks